backend/src/main/java/com/google/rolecall/services/CastServices.java

```diff
@@ -61,7 +61,9 @@ public Cast getCast(Integer id) throws EntityNotFoundException, InvalidParameter
     Cast cast = createNewCast(newCast);
     List<String> warnings = verifySubCasts(cast);
 
+    System.out.printf("AAA\n");
```

Comment:
let's remove these debugging print outs

Reply:
Sorry! I missed this because it is on the server and I didn't think I had to clean it up before doin the PR.

Removed.